packages/orchestration/test/utils/chainHub.test.ts

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable @jessie.js/safe-await-separator -- XXX irrelevant for tests */
+import test from '@endo/ses-ava/prepare-endo.js';
+
+import { makeNameHubKit } from '@agoric/vats';
+import { makeChainHub } from '../../src/utils/chainHub.js';
+
+const connection = {
+  id: 'connection-1',
+  client_id: '07-tendermint-3',
+  counterparty: {
+    client_id: '07-tendermint-2',
+    connection_id: 'connection-1',
+    prefix: {
+      key_prefix: '',
+    },
+  },
+  state: 3 /* IBCConnectionState.STATE_OPEN */,
+  transferChannel: {
+    portId: 'transfer',
+    channelId: 'channel-1',
+    counterPartyChannelId: 'channel-1',
+    counterPartyPortId: 'transfer',
+    ordering: 1 /* Order.ORDER_UNORDERED */,
+    state: 3 /* IBCConnectionState.STATE_OPEN */,
```

Comment:
Tangential to the PR - i think it would read better in vstorage if we used `ORDER_UNORDERED` and `STATE_OPEN` instead of integer

Reply:
agreed